lambdas/sqs-message-remover/tests/test-index.js

```diff
@@ -7,6 +7,12 @@ const get = require('lodash/get');
 const awsServices = require('@cumulus/aws-client/services');
 const { receiveSQSMessages } = require('@cumulus/aws-client/SQS');
 const { randomString } = require('@cumulus/common/test-utils');
+const {
+  createBucket,
+  s3PutObject,
+} = require('@cumulus/aws-client/S3');
+const { s3 } = require('@cumulus/aws-client/services');
+const { getObject } = require('@cumulus/aws-client/S3');
```

Comment:
We're already requiring `@cumulus/aws-client/S3` above, so we should add `getObject` to what is required from there